components/DiscussionsCard/DiscussionsCard.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styles from './discussionsCard.module.scss'
+import {
+  ReplyIcon,
+  ThumbsupIcon,
+  ThumbsdownIcon,
+  TrashIcon,
+  PencilIcon
+} from '@primer/octicons-react'
+
+export type DiscussionsCardProps = {
+  discussionType: 'main' | string
+  username: string
+  userPic: string
+  timeStamp: string
+  content: string
+  likes: number
+  dislikes: number
+  expandClick: Function
+  likeClick: Function
+  dislikeClick: Function
+}
+
+const DiscussionsCard: React.FC<DiscussionsCardProps> = ({
+  discussionType,
+  username,
+  userPic,
+  timeStamp,
+  content,
+  likes,
+  dislikes,
+  expandClick,
+  likeClick,
+  dislikeClick
+}) => {
+  const mainType = discussionType === 'main'
+
+  return (
+    <div className={mainType ? styles.card_body_main : styles.card_body_sub}>
+      <div
+        className={
+          mainType ? styles.card_user_info_main : styles.card_user_info_sub
+        }
+      >
+        <div>
+          <img src={userPic} className={styles.user_pic} />
+          <span className={styles.username}>{username}</span>
+          <span> </span>
+          <span> - </span>
+          <span>{timeStamp}</span>
+        </div>
+        <div className={styles.expand_button} onClick={expandClick()}>
+          ...
+        </div>
+      </div>
+      <div className={styles.card_content}>{content}</div>
+      <div className={styles.card_buttons}>
+        <div className={styles.buttons_left}>
+          <div className={styles.reply_button}>
+            <ReplyIcon /> Reply
+          </div>
+          <div className={styles.edit_delete}>
+            <PencilIcon className={styles.pencil} size={28} />
+            <TrashIcon className={styles.trash} size={28} />
+          </div>
+        </div>
+        <div className={styles.likes_container}>
+          <div className={styles.likes_button} onClick={likeClick()}>
+            <ThumbsupIcon className={styles.thumbs_up_down} />{' '}
```

Comment:
If i remember correctly, during the design meetings we agreed on not having a like/dislike button and instead use arrows and only show a sum of the votes.

In the designs we have two versions and i think we agreed on this one:
![Screen Shot 2022-09-24 at 20 17 03](https://user-images.githubusercontent.com/1410462/192114898-6d2e3272-96d4-43e8-b103-79dc5d1d055b.png)

@flacia Can you confirm this?

Reply:
We also discussed another idea where we should only have the like button:
![Screen Shot 2022-09-24 at 20 43 27](https://user-images.githubusercontent.com/1410462/192115804-2042c9bf-7b72-488d-af71-246bdbff86c9.png)

@flacia Can you confirm which one did we choose?

Reply:
@flacial Can you confirm which one did we agree on?

Reply:
I think without likes. On v2 of the discussions component, we can implement likes then.

Reply:
> I think without likes. On v2 of the discussions component, we can implement likes then.

Should we not include likes in the current PR? And add the up down arrows in a future PR?

Reply:
Or are you talking about V2 as in, after this initial discussions component has been fully implemented without likes.

Reply:
> Or are you talking about V2 as in, after this initial discussions component has been fully implemented without likes.

Yes, after this initial design.